### PR TITLE
Add tutorial how to install and configure fail2ban on debian

### PR DESCRIPTION
**Pull request by PatrickWe**
I have read and understood the Contributor's Certificate of Origin at the end of the template and I hereby certify that I meet the contribution criteria described in it.

Signed-off-by: (Patrick Weber, netcup_wiki@icloud.com)

### CODE REVIEW

File: community-tutorials/how-to-install-and-configure-fail2ban-on-debian/01-en.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+```
+
+# Step 2 - Copy the default-config file to new location
+fail2ban has two configuration files: /etc/fail2ban/jail.conf and /etc/fail2ban/jail.d/defaults-debian.conf.
+You should not update these files, because they may be overwritten when the package is updated.
+The recommended way is to create a .local file, which overrides the settings from the .conf file.
+```sudo cp /etc/fail2ban/jail.conf /etc/fail2ban/jail.local```
+
+# Step 3 - Change the [DEFAULT] section and add this configuration
+Open the configuration file ```/etc/fail2ban/jail.local``` with vi or nano.
```
Comment (d-schaffer) [inline community-tutorials/how-to-install-and-configure-fail2ban-on-debian/01-en.md:37]: -- with a texteditior

**Review by d-schaffer — COMMENTED** [review]
Thank you :)

File: community-tutorials/how-to-install-and-configure-fail2ban-on-debian/01-en.md (hunk 1 above)
Comment (PatrickWe) [reply community-tutorials/how-to-install-and-configure-fail2ban-on-debian/01-en.md]: done, thanks a lot

**Review by CoolJoni — APPROVED** [review]

**Comment by netcupClaudiaM** [comment]
Hi PatrickWe, thank you for contributing! It's been a while since you opened this PR and it probably is an open secret we're currently overwhelmed with reviewing all the awesome tutorials we got. Please be patient - we're about to publish tutorials this and next week. Thank you!

**Pull request merged**